standalone/README.md

```diff
@@ -616,6 +618,7 @@ The list of all mandatory keys:
 * `api_key`: The API key for the model serving endpoint - **Required**
 * `endpoint`: Serving endpoint for evaluation - **Required**
 * `model`: The name of the model to use for SDG - **Required**
+* `pipeline_type`: The pipeline type used for SDG, one of [simple, agentic]. - **Required**
```

Comment:
Because we use `simple` as the default, I think it's confusing to mark this field as required in the doc. I would treat this as required if we didn't have a default value. What do you think?

Reply:
yeah I agree, I've added a new commit so you can see the changes, let me know if this works

Reply:
LGTM. Let's just squash everything before we merge :)